tests/test_BasisGenerator.py

```diff
@@ -1,89 +0,0 @@
-import sys
```

Comment:
Why were these tests removed?

Reply:
These tests were removed since they were duplicated between the `test_BasisGenerator.py` and `test_pyBasisGenerator.py` files. The tests in the `test_py*` files seemed to use pytest while the others did not, so the `test_py*` versions were kept.